lib/ansible/modules/packaging/os/yum.py

```diff
@@ -146,6 +146,22 @@
         The disabled plugins will not persist beyond the transaction.
     required: false
     version_added: "2.5"
+  download_only:
```

Comment:
Can you please organise these to preserve alphabetical ordering?

Reply:
Sure thing! I was just in "append to the end" mode, but that makes sense.

Reply:
Err, actually, looking closely at this again, it seems like *nothing here* is in alphabetical order... Do you just mean these two new options? If so, I guess I can, but it seems a little bit unnecessary unless you're asking me to reorganize the whole thing.

Reply:
Sorry I didn't look at the reset of the items here. Usually the pattern I've seen across other items is to have it organised in alphabetical order.

Reply:
`ansible-doc` organizes arguments in alphabetical order so I don't think it will necessary to do here.